bpa-calculator/pom.xml

```diff
@@ -3,14 +3,13 @@
 	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
 	<modelVersion>4.0.0</modelVersion>
 	<parent>
-		<groupId>org.springframework.boot</groupId>
-		<artifactId>spring-boot-starter-parent</artifactId>
-		<version>1.5.22.RELEASE</version>
-		<relativePath /> <!-- lookup parent from repository -->
+	  <groupId>org.springframework.boot</groupId>
```

Comment:
Missing `CHANGELOG.md`, See https://github.com/egovernments/core-services/blob/5f9cd5d5af9b5d02f80302ac219226d44d792f5b/egov-idgen/CHANGELOG.md

Reply:
added change log for bpa-services and bpa-calculator with 

# 1.0.0
* Base version